package.json

```diff
@@ -148,7 +148,6 @@
     "fix:eslint": "npm run check:eslint -- --fix",
     "prepare": "node etc/prepare.js",
     "preview:docs": "ts-node etc/docs/preview.ts",
-    "release": "bash etc/check-remote.sh && standard-version -a -i HISTORY.md",
```

Comment:
should we still keep this as an emergency manual release option?

Reply:
We can always run this command ourselves, might as well do the clean up here. 

`release-please` can also be run as a CLI tool locally to generate the changelog, so I would reach for that tool first if something in the automation dies